src/Backends/Housekeeping.vala

```diff
@@ -75,18 +75,18 @@ public class SettingsDaemon.Backends.Housekeeping : GLib.Object {
     // Write (or delete) a config file in ~/.config/user-tmpfiles.d to configure the systemd timer for cleaning up the user's
     // downloads folder based on the configured age in GSettings
     private async void write_systemd_tmpfiles_config () {
-        var config_path = GLib.Path.build_filename (
-            GLib.Environment.get_user_config_dir (),
+        var config_path = Path.build_filename (
+            Environment.get_user_config_dir (),
             "user-tmpfiles.d",
             "io.elementary.settings-daemon.downloads-folder.conf"
```

Comment:
I kept the same name of the tmpfiles config file. It is technically no longer correct, as it can now contain config for screenshots dir in addition to downloads. My thinking behind this:

1. The implementation is simpler if we only have one file for all cleanup related config.
2. If we wanted to change the file name we'd need to make sure to delete the old file when we upgrade the daemon. Without that the old file would drift out of sync with gsettings and keep causing problems. This "swap files on upgrade" logic seemed like an unnecessary complication, so I kept the name the same.

Reply:
Actually, if we ship this only with OS 7 we can get away with renaming the file, as the old one won't exist on a fresh install.